fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -316,12 +305,13 @@ protected void configure() {
 
                 bind(getClock()).to(Clock.class);
 
+                DruidDimensionsLoader druidDimensionsLoader = buildDruidDimensionsLoader(
+                        nonUiDruidWebService,
+                        loader.getPhysicalTableDictionary(),
+                        loader.getDimensionDictionary()
+                );
+
                 if (DRUID_DIMENSIONS_LOADER.isOn()) {
-                    DruidDimensionsLoader druidDimensionsLoader = buildDruidDimensionsLoader(
```

Comment:
👎 If we want a hook to get at the dimensionDictionary, it would be better to just add that hook in a more long-term fashion, rather than subvert a feature flag and cause an object to be built when it's not supposed to be.